src/Controller/SponsorshipLevelBenefit/Edit.php

```diff
@@ -0,0 +1,115 @@
+    private $renderer;
+    private $sponsorshipLevelBenefitManager;
+    private $csrfTokenManager;
+
+    public function __construct(
+        Twig_Environment $renderer,
+        SponsorshipLevelBenefitManagerInterface $sponsorshipLevelBenefitManager,
+        CsrfTokenManagerInterface $csrfTokenManager
+    )
+    {
+        $this->renderer = $renderer;
+        $this->sponsorshipLevelBenefitManager = $sponsorshipLevelBenefitManager;
+        $this->csrfTokenManager = $csrfTokenManager;
+    }
+
+    /**
+     * @param FormatSponsorshipLevelBenefit $formatSponsorshipLevelBenefit
+     * @param SponsorshipLevelManagerInterface $sponsorshipLevelManager
+     * @return Response
+     * @throws \Twig_Error_Loader
+     * @throws \Twig_Error_Runtime
+     * @throws \Twig_Error_Syntax
+     */
+    public function handle(FormatSponsorshipLevelBenefit $formatSponsorshipLevelBenefit, SponsorshipLevelManagerInterface $sponsorshipLevelManager): Response
+    {
+        return new Response($this->renderer->render('sponsorshipLevelBenefit/edit.html.twig', [
+            'levels' => $sponsorshipLevelManager->getOrderedList(),
+            'levelBenefits' => $formatSponsorshipLevelBenefit->format(),
+        ]));
+    }
+
+    public function editSponsorshipLevelPosition(Request $request, SponsorshipLevelManagerInterface $sponsorshipLevelManager): JsonResponse
+    {
+        $token = new CsrfToken('sponsorship-level-benefit', $request->request->get('_token'));
+        if (!$this->csrfTokenManager->isTokenValid($token)) {
+            return new JsonResponse([false]);
+        }
+        
+        $move = $request->request->get('move');
+        $id = $request->request->get('id');
+
+        $new_level = $sponsorshipLevelManager->switchPosition($move, $id);
+
+        return new JsonResponse(['new_level_id' => $new_level->getId()]);
+    }
+
+    public function editSponsorshipBenefitPosition(Request $request, SponsorshipBenefitManagerInterface $sponsorshipBenefitManager): JsonResponse
+    {
+        $token = new CsrfToken('sponsorship-level-benefit', $request->request->get('_token'));
+        if (!$this->csrfTokenManager->isTokenValid($token)) {
+            return new JsonResponse([false]);
+        }
+
+        $move = $request->request->get('move');
+        $id = $request->request->get('id');
+
+        $new_benefit = $sponsorshipBenefitManager->switchPosition($move, $id);
+
+        return new JsonResponse(['new_benefit_id' => $new_benefit->getId()]);
+    }
+
+    public function editDatas(Request $request, SponsorshipBenefitManagerInterface $sponsorshipBenefitManager, SponsorshipLevelManagerInterface $sponsorshipLevelManager): JsonResponse
+    {
+        $token = new CsrfToken('sponsorship-level-benefit', $request->request->get('_token'));
+        if (!$this->csrfTokenManager->isTokenValid($token)) {
+            return new JsonResponse([false]);
+        }
+
+        $checked = $request->request->get('is_checked') == "false" ? false : true;
```

Comment:
single quotes unless you need string interpolation.
Also `$checked = $request->request->get('is_checked') == "false" ? false : true;` === `$checked = !($request->request->get('is_checked') == 'false');` or something like that. Please make it easier to read, why testing false? Can we test not false and say it's true then? `$checked = $request->request->get('is_checked') !== "false";`